packages/platform-sdk-ada/src/dto/transaction.ts

```diff
@@ -0,0 +1,205 @@
+import { Contracts, DTO, Exceptions } from "@arkecosystem/platform-sdk";
+import { DateTime } from "@arkecosystem/platform-sdk-intl";
+import { BigNumber } from "@arkecosystem/platform-sdk-support";
+
+export class TransactionData extends DTO.AbstractTransactionData implements Contracts.TransactionData {
+	public id(): string {
+		return this.data.id;
+	}
+
+	public blockId(): string | undefined {
+		return undefined;
+	}
+
+	public timestamp(): DateTime | undefined {
+		return DateTime.make(this.data.inserted_at.time);
+	}
+
+	public confirmations(): BigNumber {
+		return BigNumber.make(this.data.confirmations);
+	}
+
+	public sender(): string {
+		throw new Exceptions.NotImplemented(this.constructor.name, "sender");
+	}
+
+	public recipient(): string {
+		throw new Exceptions.NotImplemented(this.constructor.name, "recipient");
```

Comment:
This would cause the consumer of the package to always crash if they want to list transactions.